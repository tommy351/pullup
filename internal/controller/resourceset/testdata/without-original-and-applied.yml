---
apiVersion: pullup.dev/v1alpha1
kind: ResourceSet
metadata:
  name: test-46
  namespace: test
spec:
  number: 46
  resources:
    - apiVersion: v1
      kind: Pod
      metadata:
        name: test
      spec:
        containers:
          - name: foo
            image: gcr.io/test/foo
