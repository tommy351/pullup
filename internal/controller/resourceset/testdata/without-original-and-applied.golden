- apiVersion: v1
  kind: Pod
  metadata:
    creationTimestamp: null
    name: test-46
    namespace: test
    ownerReferences:
    - apiVersion: pullup.dev/v1alpha1
      blockOwnerDeletion: true
      controller: true
      kind: ResourceSet
      name: test-46
      uid: ""
    selfLink: /api/v1/namespaces/test/pods/test-46
  spec:
    containers:
    - image: gcr.io/test/foo
      imagePullPolicy: Always
      name: foo
      resources: {}
      terminationMessagePath: /dev/termination-log
      terminationMessagePolicy: File
    dnsPolicy: ClusterFirst
    restartPolicy: Always
    schedulerName: default-scheduler
    securityContext: {}
    terminationGracePeriodSeconds: 30
  status:
    phase: Pending
    qosClass: BestEffort
